### PR TITLE
Add support for "Auto" RC Smoothing derivative type

### PR DESCRIPTION
**Pull request by etracer65**
Related to https://github.com/betaflight/betaflight/pull/9249

### CODE REVIEW

File: src/js/tabs/receiver.js [hunk 1]
```diff
@@ -383,6 +383,11 @@ TABS.receiver.initialize = function (callback) {
             });
             rcSmoothingnDerivativeNumberElement.val(RX_CONFIG.rcSmoothingDerivativeCutoff);
             var rc_smoothing_derivative_type = $('select[name="rcSmoothingDerivativeType-select"]');
+
+            if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
+                rc_smoothing_derivative_type.append('<option value="3">' + i18n.getMessage("receiverRcSmoothingDerivativeTypeAuto") + '</option>');
```
Comment (McGiverGim) [inline src/js/tabs/receiver.js]: We now recommend to use templates to make the concatenation of strings. Is clearer. You have the info at the Sonar analysis.

Comment (etracer65) [reply src/js/tabs/receiver.js]: Not really a Javascript programmer. Can you give me an example? The code style I used was just a copy/paste from another spot in the same file.

Comment (mikeller) [reply src/js/tabs/receiver.js]: MDN is often the go-to place for clear and accurate information for JavaScript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

TL;DR: They work the same way as string interpolation does in C#. :trollface:

Comment (McGiverGim) [reply src/js/tabs/receiver.js]: I suggested to look into Sonar because it shows a sample with each issue 😋

Comment (mikeller) [reply src/js/tabs/receiver.js]: @McGiverGim: Yeap, Sonar is good too as in they provide helpful hints for the stuff they detect.

Comment (McGiverGim) [reply src/js/tabs/receiver.js]: @etracer65 you have the info here: https://sonarcloud.io/project/issues?id=betaflight_betaflight-configurator&pullRequest=1831&resolved=false&types=CODE_SMELL

Comment (etracer65) [reply src/js/tabs/receiver.js]: Updated the concatenation style.

Comment (etracer65) [reply src/js/tabs/receiver.js]: Gotta say though, I don't really see how
```
$(`<option value="3">${i18n.getMessage("receiverRcSmoothingDerivativeTypeAuto")}</option>`)
```
is more "maintainable" than
```
'<option value="3">' + i18n.getMessage("receiverRcSmoothingDerivativeTypeAuto") + '</option>'
```

Comment (mikeller) [reply src/js/tabs/receiver.js]: @etracer65: You are not looking hard enough at (the madness of) JavaScript's type coercion. try `'11' + 1 - 2 + ' '``...
